runtime/peaq-dev/src/lib.rs

```diff
@@ -960,11 +961,27 @@ impl zenlink_protocol::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
```

Comment:
I think we should set this to mirror peaq-runtime also

Reply:
After discuss with Karman, the peaq-dev's inflation will follow the krest network's setting